bulk-update/bulk-update.js

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs';
+import { fetch } from '@adobe/fetch';
+import { loadDocument, saveDocument } from './document-manager/document-manager.js';
+
+export async function loadFromUrl(url) {
+  const response = fetch(url);
+  return response.json();
+}
+
```

Comment:
Do you need an await on the response?

Reply:
That's right, I need to resolve the promise or return the promise chain, since async always returns a promise I can remove async for the same result.